modules/test/playwright/utils/fillAndClickOutside.ts

```diff
@@ -10,9 +10,13 @@ import getRandomString from './getRandomString';
 export default async function fillAndClickOutside(
 	page: Page,
 	element: Locator,
-	content?: string
+	content?: string,
+	clickOutside: boolean = true
```

Comment:
should be sorted

Reply:
Yep, no need actually. I misunderstood the context when I added that comment. For JS we don't always require parameters of a function to be sorted. Thanks.